integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/metadata/MetadataConverter.java

```diff
@@ -0,0 +1,53 @@
+package se.inera.intyg.minaintyg.integration.webcert.converter.metadata;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.CertificateMetadata;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.common.CertificateIssuer;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.common.CertificateType;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.common.CertificateUnit;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.CertificateMetadataDTO;
+
+@Component
+@RequiredArgsConstructor
+public class MetadataConverter {
+
+  private final EventConverter eventConverter;
+  private final StatusConverter statusConverter;
+
+  public CertificateMetadata convert(CertificateMetadataDTO metadataDTO) {
+    return CertificateMetadata.builder()
+        .id(metadataDTO.getId())
+        .type(convertType(metadataDTO))
+        .issuer(convertIssuer(metadataDTO))
+        .unit(convertUnit(metadataDTO))
+        .events(eventConverter.convert(metadataDTO))
+        .statuses(statusConverter.convert(metadataDTO))
+        .issued(metadataDTO.getCreated())
+        .build();
```

Comment:
Add CertificateRecipient to the conversion. It was added in the "send certificate"-story so you can now add it to this converter as well.